### ISSUE TITLE
Linux Meshagent fails if invite is from sub-domain (cusotmer)

### ISSUE THREAD

**Issue by Kodangi** [Ylianst/MeshCentral#225]
Linux Meshagent (meshinstall.sh) fails if invite is from sub-domain (customer1 for example) with error 'Username/Password Authentication Failed'

Windows Agent works fine under sub-domain (customer1)

**Comment by Ylianst** [comment]
Ha! Thanks for the report, I will need to look into this.

**Comment by Ylianst** [comment]
Hi. I just tested sending an invitation email and cut & pasting the install script on the Linux device and it worked for me. My sub-domain was "customer1" and my test shell script looked like this:

```
wget -q https://devbox.mesh.meshcentral.com:443/customer1/meshagents?script=1 --no-check-certificate -O ./meshinstall.sh && chmod 755 ./meshinstall.sh && sudo ./meshinstall.sh https://devbox.mesh.meshcentral.com:443/customer1 EaSw1hJBKjf1QApmilFSAIo1KhA@R@dgRHIDAUUZBaMBGEYJw9qlhOa7p0xfWZvN
```

The agent install correctly and the content of /usr/local/mesh/meshagent.msh looked like this:

```
MeshName=TestGroup
MeshType=2
MeshID=0x11A4B0D612412A37F5400A668A5152008A352A103E47E76044720301451905A301184609C3DAA584E6BBA74C5F599BCD
ServerID=D99362D5ED8BAEA8BF9E743B34B242256370C460FD66CB62373C6CFCB204D6D707403E396CF0EF6DC2B3A42F735135FD
MeshServer=wss://devbox.mesh.meshcentral.com:443/customer1/agent.ashx
StartupType=1
```

Note the "MeshServer" line with "customer1" in it. Can you try again and send more details on what is going wrong. You may need to X-out your server name or send me a mail directly if needed. Thanks.

**Comment by Kodangi** [comment]
Thanks Ylian, your example pointed me the problem immediately. The URL I get from 'Add Agent' does not have 'customer1'. 

I manually added and ran, it went fine.


When I debug, I tried changing config.json --> domains --> customer1 --> DNS to www.company.com/customer1, then I was not able to login to custome1 site. So I removed the entry DNS in customer1, then i was able to login. 
But the 'Add Agent' is providing the URL without the customer name in it.

I am OK now, but general users expect the 'Add Agent' to provide the URL with customer1 on it.
